<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeitstellenWindow</class>
 <widget class="QMainWindow" name="LeitstellenWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LeitstellenWindow</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
    <rect>
        <x>0</x>
        <y>0</y>
        <width>450</width>
        <height>500</height>
    </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
        <item>
            <spacer name="verticalSpacer">
            <property name="orientation">
                <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
            <size>
                <width>20</width>
                <height>40</height>
            </size>
            </property>
            </spacer>
        </item>
    <item>
    <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
        <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>481</height>
        </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
        
        <item>
            <spacer name="horizontalSpacer_1">
            <property name="orientation">
            <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
            <size>
                <width>20</width>
                <height>40</height>
            </size>
            </property>
            </spacer>
        </item>

        <item>
            <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                    <string>Abgeschlossene Einsätze</string>
                </property>
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="minimumSize">
                    <size>
                        <width>400</width>
                        <height>500</height>
                    </size>
                </property>
                <widget class="QListWidget" name="alteEinsaetze">
                    <property name="minimumSize">
                        <size>
                            <width>400</width>
                            <height>455</height>
                        </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>20</pointsize>
                     </font>
                    </property>
                    <property name="geometry">
                        <rect>
                            <x>0</x>
                            <y>45</y>
                        </rect>
                    </property>
                </widget>
                <!-- <widget class="QPlainTextEdit" name="alteEinsaetze">
                    <property name="minimumSize">
                        <size>
                            <width>400</width>
                            <height>455</height>
                        </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>20</pointsize>
                     </font>
                    </property>
                    <property name="geometry">
                        <rect>
                            <x>0</x>
                            <y>45</y>
                        </rect>
                    </property>
                </widget> -->
            </widget>
        </item>
        
        <item>
            <spacer name="horizontalSpacer_2">
            <property name="orientation">
            <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
            <size>
                <width>20</width>
                <height>40</height>
            </size>
            </property>
            </spacer>
        </item>

        <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Aktuelle Einsätze</string>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="minimumSize">
                 <size>
                     <width>400</width>
                     <height>500</height>
                 </size>
             </property>
             <widget class="QListWidget" name="aktuelleEinsaetze">
                <property name="minimumSize">
                    <size>
                        <width>400</width>
                        <height>455</height>
                    </size>
                </property>
                <property name="toolTip">
                 <string>Drücke ENTF, um die markierten Einsätze zu löschen.</string>
                </property>
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>45</y>
               </rect>
              </property>
             </widget>
             <!-- <widget class="QPlainTextEdit" name="aktuelleEinsaetze">
                <property name="minimumSize">
                    <size>
                        <width>400</width>
                        <height>455</height>
                    </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>45</y>
               </rect>
              </property>
             </widget> -->
            </widget>
        </item>
        
        <item>
            <spacer name="horizontalSpacer_3">
            <property name="orientation">
            <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
            <size>
                <width>20</width>
                <height>40</height>
            </size>
            </property>
            </spacer>
        </item>

        <item alignment="Qt::AlignVCenter">
            <widget class="QPushButton" name="pushButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>150</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>23</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Neuer Einsatz</string>
             </property>
             <property name="font">
              <font>
               <pointsize>25</pointsize>
              </font>
             </property>
            </widget>
        </item>

        
        <item>
            <spacer name="horizontalSpacer_4">
            <property name="orientation">
            <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
            <size>
                <width>20</width>
                <height>40</height>
            </size>
            </property>
            </spacer>
        </item>

    </layout>
</widget>
</item>
<item>
    <spacer name="verticalSpacer_2">
    <property name="orientation">
        <enum>Qt::Vertical</enum>
    </property>
    <property name="sizeHint" stdset="0">
    <size>
        <width>20</width>
        <height>40</height>
    </size>
    </property>
    </spacer>
</item>
</layout>
</widget>
</widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
